### ISSUE TITLE
Inconsistent caster ID's

### ISSUE THREAD

**Issue by boada** [schlaf/schlaf.github.io#13]
There is a mix of upper and lower case in the caster ID's (I bet in other places too). The naming isn't consistent across factions and in some cases isn't consistent within factions. I don't want to change it, as I don't know how it will affect the theme forces and other things within the app.

**Comment by schlaf** [comment]
yes, this is so-so, but not too critical... actually, making everything consistent will make old lists buggy (will not load anymore) and will prevent importing old lists, and will cause duplicate if using older datafiles...

as those ids are purely technical (never appear) i think i will let them as is unless there is some major change needed major refactoring of data.

**Comment by boada** [comment]
What convention do we want to use? My vote is for all uppercase.
